3-reveal/migrations/5-IRS/4-Zambia-2020/deploy/remove_hard_coded_jurisdiction_depth.psql

```diff
@@ -98,27 +98,23 @@ DROP MATERIALIZED VIEW IF EXISTS pending_:schema .zambia_irs_structures CASCADE;
 
 CREATE MATERIALIZED VIEW IF NOT EXISTS pending_:schema .zambia_irs_structures
 AS
-SELECT DISTINCT ON (locations.id, events_query.task_id)
+SELECT DISTINCT ON (irs_plans.plan_id, locations.id)
     public.uuid_generate_v5(
         '6ba7b810-9dad-11d1-80b4-00c04fd430c8',
-        concat(locations.id, events_query.task_id)) AS id,
+        concat(locations.id, COALESCE(events_query.task_id, tasks_query.identifier))) AS id,
+    irs_plans.plan_id,
     locations.id AS structure_id,
+    locations.status AS structure_status,
     locations.jurisdiction_id AS old_jurisdiction_id,
     zambia_structure_jurisdictions.jurisdiction_depth AS old_jurisdiction_depth,
     zambia_structure_jurisdictions.geo_jurisdiction_id AS geo_jurisdiction_id,
     zambia_structure_jurisdictions.geo_jurisdiction_depth AS geo_jurisdiction_depth,
     zambia_structure_jurisdictions.zambia_jurisdiction_id AS zambia_jurisdiction_id,
     zambia_structure_jurisdictions.zambia_jurisdiction_depth AS zambia_jurisdiction_depth,
     COALESCE(zambia_structure_jurisdictions.zambia_jurisdiction_id, locations.jurisdiction_id) AS structure_jurisdiction_id,
-    locations.code AS structure_code,
-    locations.name AS structure_name,
-    locations.type AS structure_type,
-    locations.geometry AS structure_geometry,
-    locations.status AS structure_status,
```

Comment:
These fields are not really needed on this materialized view.  They can be joined in at the final stage that is exposed to Superset/the web app.